workers/generic-worker/main.go

```diff
@@ -81,10 +81,10 @@ func initialiseFeatures() (err error) {
 	}
 	features = append(features, platformFeatures()...)
 	for _, feature := range features {
-		log.Printf("Initialising task feature %v...", feature.Name())
```

Comment:
I agree with this cleanup, thanks. 👍🏻 Since not all features are _task_ features.